pyproject.toml

```diff
@@ -64,6 +64,7 @@ zeep = {version = "^4.1.0", optional = true}
 snowflake-connector-python = {version = "^2.7.12", optional = true}
 pyarrow = {version = "<9", optional = true}
 toucan-client = {version = "^1.0.1", optional = true}
+peakina = {version = "^0.9.1", extras = ["all"]}
```

Comment:
I think that for consistency, this should also be an optional extra

Reply:
https://github.com/ToucanToco/toucan-connectors/pull/838/commits/3f306c36c0203867952200ef48ac0e1799d93765